combat-trainer.lic

```diff
@@ -2153,7 +2161,9 @@ class GameState
   end
 
   def melee_attack_verb
-    if attack_override
+    if @use_tactics_combos && !offhand? && brawl_tactics? && !backstab? && tactics_analyze?
+      @tactics_combo_array.shift
+    elsif attack_override
       attack_override
```

Comment:
It feels to me like if the user has an override specified, it should take higher priority than completing a combo. What do you think?